qt/python/mantidqt/project/plotssaver.py

```diff
@@ -40,7 +40,10 @@ def save_plots(self, plot_dict):
                 # have built your project.
                 if isinstance(e, KeyboardInterrupt):
                     raise KeyboardInterrupt
-                logger.warning("A plot was unable to be saved")
+                if not is_project_recovery:
+                    logger.warning("A plot was unable to be saved")
```

Comment:
Can we include some identifier of the plot, such as the name, in the error message?

Reply:
It's not guaranteed there will be a name for the plot due to the nature of how some graphs are generated.

Reply:
Can we at least put the index of the plot, and append the `str(e)`?

Reply:
I'll see what I can do

Reply:
Changes made :+1: